src/Core/Localization/Locale.php

```diff
@@ -160,7 +160,7 @@ public function formatPrice($number, $currencyCode)
      *
      * @param string $currencyCode Currency of the price
      *
-     * @return NumberSpecificationInterface
+     * @return NumberSpecificationInterface|null
```

Comment:
I don't think these ones are nullable, they are provided in the constructor and should always be present

Reply:
```suggestion
     * @return NumberSpecificationInterface
```

if it's null, the method triggers an exception

Reply:
Exactly so it can never return null in the end Since it's not in the PR's scope it's worth fixing this one before messing with the codebase of CLDR ^^